### PR TITLE
Support sepa_debit for bancontact, ideal, sofort

### PR DESCRIPTION
**Pull request by richardm-stripe**

### CODE REVIEW

File: paymentmethod.go [hunk 1]
```diff
@@ -327,13 +327,21 @@ type PaymentMethodOXXO struct {
...
-	BranchCode  string `json:"branch_code"`
-	Country     string `json:"country"`
-	Fingerprint string `json:"fingerprint"`
-	Last4       string `json:"last4"`
+	BankCode      string                              `json:"bank_code"`
+	BranchCode    string                              `json:"branch_code"`
+	Country       string                              `json:"country"`
+	Fingerprint   string                              `json:"fingerprint"`
+	Last4         string                              `json:"last4"`
+	GeneratedFrom PaymentMethodSepaDebitGeneratedFrom `json:"generated_from"`
```
Comment (cjavilla-stripe) [inline paymentmethod.go:344]: should this one also be a pointer?

Comment (richardm-stripe) [reply paymentmethod.go:344]: Yup good catch.

**Comment by richardm-stripe** [comment]
ptal @cjavilla-stripe

**Review by cjavilla-stripe — APPROVED** [review]
LGTM

**Pull request merged**